Project.toml

```
@@ -13,6 +14,7 @@ TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
 
 [compat]
 ArgParse = "1.1"
+EllipsisNotation = "1.0.0"
```

```suggestion
EllipsisNotation = "1"
```
should be sufficient and a bit less verbose